src/test/java/org/fogbowcloud/manager/requests/api/local/http/AttachmentOrdersControllerTest.java

```diff
@@ -0,0 +1,395 @@
+import org.fogbowcloud.manager.core.exceptions.InstanceNotFoundException;
+import org.fogbowcloud.manager.core.models.InstanceStatus;
+import org.fogbowcloud.manager.core.models.instances.AttachmentInstance;
+import org.fogbowcloud.manager.core.models.instances.InstanceState;
+import org.fogbowcloud.manager.core.models.instances.InstanceType;
+import org.fogbowcloud.manager.core.models.orders.AttachmentOrder;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import junit.framework.Assert;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockRunnerDelegate(SpringRunner.class)
+@WebMvcTest(value = AttachmentOrdersController.class, secure = false)
+@PrepareForTest(ApplicationFacade.class)
+public class AttachmentOrdersControllerTest {
+    private final String ATTACHMENT_ENDPOINT =
+            "/".concat(AttachmentOrdersController.ATTACHMENT_ENDPOINT);
+
+    private final String CORRECT_BODY =
+            		"{"
+                    + "\"source\": \"b8852ff6-ce00-45aa-898d-ddaffb5c6173\","
+                    + "\"target\": \"596f93c7-06a1-4621-8c9d-5330a089eafe\","
+                    + "\"device\": \"/dev/sdd\""
+                    + "}";
+
+    private final String BODY_WITH_EMPTY_PROPERTIES =
+            		"{"
+                    + "\"source\": \"b8852ff6-ce00-45aa-898d-ddaffb5c6173\","
+                    + "\"target\": \"596f93c7-06a1-4621-8c9d-5330a089eafe\","
+                    + "\"device\": \"/dev/sdd\""
+                    + "}";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    private ApplicationFacade facade;
+
+    @Before
+    public void setUp() throws FogbowManagerException {
+        this.facade = Mockito.spy(ApplicationFacade.class);
+        PowerMockito.mockStatic(ApplicationFacade.class);
+        given(ApplicationFacade.getInstance()).willReturn(this.facade);
+    }
+
+    // test case: Request a attachment creation and test successfully return. 
+    // Check the response of request and the call of facade for create the attachment.
```

Comment:
I think you can omit this second line in your test cases. Keep only the description of the case, not how we assert stuff